src/test/java/com/cgdp/library/validation/ValidatorTest.java

```diff
@@ -0,0 +1,175 @@
+    }
+
+    @Test
+    void should_throw_exception_when_the_value_is_null() {
+        //given
+        String paramName = "value";
+        String value = null;
+
+        //when
+        Throwable thrownException = catchThrowable(
+              () -> Validator.requiredNotBlank(paramName, value));
+
+        //then
+        assertThat(thrownException)
+              .isInstanceOf(ValidationException.class)
+              .hasMessage(String.format("%s must not be null or blank", paramName));
+    }
+
+    @Test
+    void should_throw_exception_when_date_value_is_after_now() {
+        //given
+        String paramName = "value";
+        LocalDate value = LocalDate.now().plusDays(1);
+
+        //when
+        Throwable thrownException = catchThrowable(
+              () -> Validator.requiredBeforeNow(paramName, value));
+
+        //then
+        assertThat(thrownException)
+              .isInstanceOf(ValidationException.class)
+              .hasMessage(String.format("Invalid %s. %s is after current date", paramName, value));
+    }
+
+    @Test
+    void should_return_date_value_before_now() {
+        // given
+        String paramName = "value";
+        LocalDate value = LocalDate.now().minusDays(1);
+
+        // when
+        LocalDate result = Validator.requiredBeforeNow(paramName, value);
+
+        // then
+        assertThat(result).isEqualTo(value);
+    }
+
+    @Test
+    void should_throw_exception_when_isbn_value_is_invalid() {
+        //given
+        String paramName = "value";
+        String value = "9780134685992";
+
+        //when
+        Throwable thrownException = catchThrowable(
+              () -> Validator.requiredValidIsbn(paramName, value));
+
+        //then
+        assertThat(thrownException)
+              .isInstanceOf(ValidationException.class)
+              .hasMessage(String.format(
+                    "%s is not a valid ISBN. An ISBN must have 13 numbers and have a valid check number",
+                    value));
+    }
+
+    @Test
+    void should_throw_exception_when_isbn_value_is_empty() {
+        //given
+        String paramName = "value";
+        String value = "";
+
+        //when
+        Throwable thrownException = catchThrowable(
+              () -> Validator.requiredValidIsbn(paramName, value));
+
+        //then
+        assertThat(thrownException)
+              .isInstanceOf(ValidationException.class)
+              .hasMessage(String.format(
+                    "%s is not a valid ISBN. An ISBN must have 13 numbers and have a valid check number",
+                    value));
+    }
+
+    @Test
+    void should_return_valid_isbn() {
+        // given
+        String paramName = "value";
+        String value = "9780134685991";
+
+        // when
+        String result = Validator.requiredValidIsbn(paramName, value);
+
+        // then
+        assertThat(result).isEqualTo(value);
+    }
+
+    @Test
+    void should_return_value_when_isbn_value_is_not_empty() {
```

Comment:
What is the difference between this test and the one above? `should_return_valid_isbn`